src/tools/testmanagement-utils/list-testcases.ts

```diff
@@ -0,0 +1,104 @@
+import axios from "axios";
+import config from "../../config";
+import { z } from "zod";
+import { CallToolResult } from "@modelcontextprotocol/sdk/types.js";
+import { formatAxiosError } from "../../lib/error";
+
+/**
+ * Schema for listing test cases with optional filters.
+ */
+export const ListTestCasesSchema = z.object({
+  project_identifier: z
+    .string()
+    .describe("Identifier of the project to fetch test cases from."),
```

Comment:
Add example identifier here.

```suggestion
    .describe("Identifier of the project to fetch test cases from. Example: ...."),
```